### ISSUE TITLE
ilasm/ildasm/ilverify as dotnet global tools

### ISSUE THREAD

**Issue by am11** [dotnet/coreclr#27934]
For example, acquisition and usage of https://www.nuget.org/packages/dotnet-format/ is quite simple:

```
dotnet tool install --global dotnet-format
```

```
dotnet-format --files MyModel.cs
```

It would be handy if IL-related tools are also packaged as [.NET Core Global tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create) and following the installation instructions on nuget.org page (`dotnet tool install ..`) is sufficient.

**Comment by danmosemsft** [comment]
Ildasm (at least) is Windows specific right now. If there were global tools ideally they’d be cross platform.

@safern didn’t you mention you wrote a cross platform ildasm?

**Comment by am11** [comment]
With the current master, it seems to be building on Unix, e.g.:

https://github.com/dotnet/coreclr/blob/a9f3fc16483eecfc47fb79c362811d870be02249/src/ildasm/exe/CMakeLists.txt#L12-L18

we get `coreclr/bin/Product/Linux.x64.Debug/ildasm` executable.

**Comment by jkotas** [comment]
> Ildasm (at least) is Windows specific right now

ildasm is not Windows specific. Did you mean to say platform specific?

**Comment by danmosemsft** [comment]
@jkotas I meant it’s a Win32 UI app isn’t it? At least the original one I’m used to.

**Comment by jkotas** [comment]
The .NET Framework one is a dual command line tool and Win32 UI app. The .NET Core one is a regular (unmanaged) command line tool. We did not bother to open source the Win32 UI.

**Comment by MarcoRossignoli** [comment]
@jkotas ILVerify is the one in corert repo?Will be moved to new repo?

**Comment by jkotas** [comment]
Yes, I think it would make sense. Opened https://github.com/dotnet/coreclr/issues/27939
